wezterm-gui/src/overlay/quickselect.rs

```diff
@@ -660,14 +678,6 @@ impl QuickSelectRenderable {
                             let mut r = search_overlay.renderer.borrow_mut();
                             r.results = results.take().unwrap();
                             r.recompute_results();
-                            let num_results = r.results.len();
-
-                            if !r.results.is_empty() {
```

Comment:
Removing this code changes the behavior for the case where no range is specified.
What's your rationale for this? Does it make sense to only skip this when range(s) were passed in?
Should this be a separate argument for quickselect?